### ISSUE TITLE
Compilation errors

### ISSUE THREAD

**Issue by tobigithub** [PNNL-Comp-Mass-Spec/LIQUID#1]
Just pulled from the repo (via ZIP donwload) and compiled in VS 2015,
I got three errors for the X64 target. Maybe check clean compile in
a virtual machine, so no dependency issues will occur on "new" computers.

```
Severity	Code	Description	Project	File	Line	Suppression State

Warning		Could not resolve this reference. Could not locate the assembly "log4net". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.	LiquidBackend			

Error	CS0246	The type or namespace name 'Mage' could not be found (are you missing a using directive or an assembly reference?)	LiquidBackend	X:\LIQUID\LiquidBackend\Scoring\ScoreModelCreator.cs	10	Active

Error	CS0246	The type or namespace name 'Mage' could not be found (are you missing a using directive or an assembly reference?)	LiquidBackend	X:\LIQUID\LiquidBackend\Util\DmsDatasetFinder.cs	6	Active

Error	CS0006	Metadata file 'X:\LIQUID\LiquidBackend\bin\x64\Release\LiquidBackend.dll' could not be found	Liquid	X:\LIQUID\Liquid\CSC	1	Active

Warning		The referenced component 'Mage' could not be found.	LiquidBackend			
Warning		Could not resolve this reference. Could not locate the assembly "Mage". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.	LiquidBackend			
Warning		The referenced component 'log4net' could not be found.	LiquidBackend			
Warning		The referenced component 'MyEMSLReader' could not be found.	LiquidBackend			
Warning		Could not resolve this reference. Could not locate the assembly "MyEMSLReader". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.	LiquidBackend			

```

**Comment by alchemistmatt** [comment]
Thank you for reporting this.  The DLLs were already in the Repo, but the HintPath in the C# project file wasn't correct.  I have fixed this in commit 09c5f0390

**Comment by tobigithub** [comment]
Perfect, thank you.
Compiles are correct now.

````
1>------ Rebuild All started: Project: LiquidBackend, Configuration: Release x64 ------
1>  LiquidBackend -> Z:\LIQUID-master\LiquidBackend\bin\x64\Release\LiquidBackend.dll
2>------ Rebuild All started: Project: Liquid, Configuration: Release x64 ------
3>------ Rebuild All started: Project: LiquidTest, Configuration: Release x64 ------
2>  Liquid -> Z:\LIQUID-master\Liquid\bin\x64\Release\Liquid.exe
3>  LiquidTest -> Z:\LIQUID-master\LiquidTest\bin\x64\Release\LiquidTest.dll
========== Rebuild All: 3 succeeded, 0 failed, 0 skipped ==========
````
